ocs_ci/deployment/aws.py

```diff
@@ -291,14 +291,21 @@ class OCPDeployment(BaseOCPDeployment):
         def __init__(self):
             super(AWSUPI.OCPDeployment, self).__init__()
             upi_repo_name = f'openshift-misc-{config.RUN["run_id"]}'
+            self.ocp_version = ".".join(
+                config.DEPLOYMENT.get('installer_version').split('.')[:-2]
```

Comment:
I see this: config.DEPLOYMENT.get('installer_version').split('.')[:-2] on 2 places. If I remember correctly you said that you just copy pasted this from some other place as well.

WDYT about have some func in utils like get_ocp_version() which will return this value and you will replace also on other places where it is used as well?

Maybe you can add also parameter like separator and in case it's defined you can replace for example with this separator like _ so it will return "4_2" which you can use on different places where you do replace

Reply:
Done, introduced a new function get_ocp_version().